pkg/appmanager/routing.go

```diff
@@ -394,7 +394,7 @@ func updateDataGroupForPassthroughRoute(
 	dgMap InternalDataGroupMap,
 ) {
 	hostName := route.Spec.Host
-	poolName := formatRoutePoolName(route)
+	poolName := formatRoutePoolName(route, route.Spec.To.Name) // TODO(kenr): Do we need to make this svcName? (same below)
```

Comment:
I think we would want this to be svcName. If a Route is going to specify more than one backend, then we want the passthrough/reencrypt rules to apply to any pools for that route. So the data group would need to know about each pool for the Route.

Reply:
It may be worth getting @f5gary's input on this as well.